.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: php
+
+sudo: false
+
```

Comment:
I suggest also adding the cache for composer downloads, and for the simple-phpunit downloads of PHPUnit):

```
cache:
  directories:
    - $HOME/.composer/cache/files
    - $HOME/symfony-bridge/.phpunit

env:
  global:
    - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"